src/wizard.scss

```diff
@@ -0,0 +1,300 @@
+      .#{$block}__step-indicator {
+        background: var(--sapObjectHeader_Background);
+        border: var(--sapWizard_Upcoming_Step_Border);
+        color: var(--sapWizard_Upcoming_Step_Color);
+
+        &::before {
+          color: var(--sapWizard_Upcoming_Step_Icon_Color);
+        }
+      }
+
+      .#{$block}__step-container {
+        cursor: default;
+        pointer-events: none;
+      }
+    }
+
+    &--no-label {
+      min-width: $fd-wizard-step-no-label-max-size;
+
+      .#{$block}__label-container {
+        display: none;
+      }
+    }
+
+    &--stacked-top {
+      &.#{$block}__step--stacked {
+        min-width: $fd-wizard-step-no-label-max-size;
+        width: $fd-wizard-step-no-label-max-size;
+
+        .#{$block}__connector {
+          display: block;
+          min-width: auto;
+        }
+      }
+    }
+
+    &--stacked {
+      width: 0.25rem;
+      min-width: auto;
+
+      .#{$block}__connector,
+      .#{$block}__label-container {
+        display: none;
+      }
+
+      &.#{$block}__step--upcoming {
+        &:last-child {
+          padding-right: $fd-wizard-stacked-upcoming-step-offset;
+
+          @include fd-rtl() {
+            padding-right: 0;
+            padding-left: $fd-wizard-stacked-upcoming-step-offset;
+          }
+        }
+
+        // Fiori 3 recommends a maximum of 8 steps
+        @for $i from 1 through 8 {
+          &:nth-child(#{$i}) {
+            z-index: calc(10 - #{$i});
+          }
+        }
+      }
+    }
+
+    &--active {
+      .#{$block}__step-indicator {
+        background: var(--sapActiveColor);
+        border-color: var(--sapWizard_Active_Step_Border_Color);
+        color: var(--sapWizard_Active_Step_Color);
+      }
+
+      &::before {
+        color: var(--sapWizard_Active_Step_Icon_Color);
+      }
+    }
+  }
+
+  &__step-container {
+    @include fd-reset();
+    @include fd-flex-vertical-center();
+
+    @include fd-focus() {
+      outline-style: var(--sapContent_FocusStyle);
+      outline-width: var(--sapContent_FocusWidth);
+      outline-color: var(--sapContent_FocusColor);
+    }
+
+    cursor: pointer;
+  }
+
+  &__step-indicator {
+    @include fd-reset();
+    @include fd-flex-center();
+
+    border-radius: 50%;
+    min-width: $fd-wizard-step-indicator-size;
+    min-height: $fd-wizard-step-indicator-size;
+    width: $fd-wizard-step-indicator-size;
+    height: $fd-wizard-step-indicator-size;
+    margin: 0 $fd-wizard-step-indicator-spacing;
+
+    &::before {
+      font-size: var(--sapFontLargeSize);
+    }
+  }
+
+  &__label-container {
+    @include fd-reset();
+    @include fd-flex(column);
+
+    max-width: 100%;
+    max-height: $fd-wizard-step-indicator-size;
+    color: var(--sapContent_LabelColor);
+    overflow: hidden;
+
+    &--optional {
+      .#{$block}__label {
+        @include line-clamp(1);
+      }
+    }
+  }
+
+  &__label {
+    @include fd-reset();
+    @include line-clamp();
+
+    line-height: 1;
```

Comment:
IE11
![image](https://user-images.githubusercontent.com/26483208/93210315-ad51e280-f75f-11ea-996b-465b0102e4b9.png)

Reply:
This is ok :) we are using line-clamp and it's not working in IE11. I can check the line-height, but the rest can't be fixed (as we agreed and discussed multiple times when we decided to go with line-clamp)